a-braham/Questioner
# #163347238 Modified profile page

#### What does this PR do?
 * The pull request modifies profile.html page

#### Description of Task to be completed?
  **Have the following features complete**
  * Responsiveness of the profile page
  * Should not have any inline css

#### How should this be manually tested?
  * Clone the repository
  * Open the /profile.html files with chrome
  * Test for responsiveness, check the html tags and css for errors
 
#### What are the relevant pivotal tracker stories?
  #163347238

#### Screenshots
![screenshot from 2019-01-20 14-32-41](https://user-images.githubusercontent.com/45788436/51438684-5f51b680-1cc0-11e9-89d3-2b9c61f93903.png)
